src/applications/letters/tests/01-authed.e2e.spec.js

```diff
@@ -22,11 +22,26 @@ module.exports = E2eHelpers.createE2eTest(client => {
     .expect.element('.usa-accordion-bordered')
     .to.be.present.before(Timeouts.normal);
 
+  client.elements('css selector', '.usa-accordion-bordered', result => {
+    client.assert.equal(result.value.length, 5);
+  });
```

Comment:
Oh cool, I couldn't get the `elements` method to work but now I see I wasn't adding that `'css selector'` part.

Reply:
Yeah, nightwatch is new to me too 🤣